lib/topological_inventory/schema/default.rb

```diff
@@ -37,6 +37,15 @@ def initialize_inventory_collections
         add_cross_link_vms
       end
 
+      def to_hash
+        {
+          "schema": {
+            "name": "Default"
```

Comment:
Can we get the schema name dynamically that way if we have other schemas that inherit from this one they can just the same method?

Reply:
I think we can do that once we have some ind of base schema?

Reply:
or should I just put that to persister?

Reply:
No I think here makes sense, so can you add a `TopologicalInventory::Schema::Base` with this there?  Not sure if any of the other helper methods make sense in base but some might.